developer-docs/latest/concepts/webhooks.md

```diff
@@ -299,6 +299,10 @@ This event is triggered only when you delete a media through the media interface
 
 ## Developer documentation
 
+### User content type webhooks
+To prevent from unintentionally sending any user's information to other applications, Webhooks will not work for the User content type.
+If you need to notify other applications about changes in the Users collection, you can do so by creating [Lifecycle hooks](https://strapi.io/documentation/developer-docs/latest/concepts/models.html#lifecycle-hooks) inside the file `./extensions/users-permissions/models/User.js`.
```

Comment:
Can you please use relative links to the file?

Reply:
Hi @derrickmehaffy thank you for pointing that out. I think it's fixed now in 7a2ce86405007ce2558e174aea8dce8c25a4d42f

Please, let me know if I need to finx anything else